STYLE_GUIDE.md

```diff
@@ -240,6 +240,25 @@ rescue MyException => error
   # do something
 end
 ```
+
+
+* <a name="style-empty-method"></a>
+  Write empty methods in expanded way.
+  <sup>[[link](#empty-method)]</sup>
+
+```ruby
+# bad
+def foo(bar); end
+
+def self.foo(bar); end
+
+# good
+def foo(bar)
+end
+
+def self.foo(bar)
+end
+```
 ## Rspec
```

Comment:
Can we have one-line empty space before this header? like for others https://github.com/datarockets/datarockets-style/pull/59/files#diff-1550192a99adeea1f5b641e7576116b7R13